README.md

```diff
@@ -1 +1,28 @@
-# hw01
\ No newline at end of file
+
+
+## Name: Mengdi Huang
+* *University of Chicago*
+* **MAPSS**
+* My research interests:
+1. Social Cognition
+1. Emotions
+1. Behavioral Science
+
+* I got interested in psychology when I was in high school, when I was a volunteer teaching assistant in an autism school. I interacted with a little patient called Dongdong. Though he was very reluctant to interact with me at first, I became his favorite teacher in the end. This experience made me wonder that why patients with autism are often emotionally unstable and what I can do for them. Therefore I decided to study psychology during my undergraduate life.
+
+* Aside from psychology, here is a little fun facts about me:
+Born and raised in a small tropical island in China, I chose to study abroad when I was 18 and I completed my undergraduate in California, San Diego, where it has ocean, palm trees and amazing tacos. So this is my first time experiencing actual winter that snows and I hope that I will survive...I also love traveling around, walking in the afternoon and listening to chill lofi music on youtube. My favorite drink is boba :)
+
+
+
+
+*My favorite animal is cat (Though I am a dog person)
+
+
+
+
+
+
+![Here's a picture of me trying to program in R](download-1.jpg)
```

Comment:
Mengdi, 

Your image was embedded nicely in RMarkdown. Your image was creative and funny! I also love cats!